packages/plugins/users-permissions/server/graphql/mutations/crud/user/update-user.js

```diff
@@ -25,7 +25,7 @@ module.exports = ({ nexus, strapi }) => {
       data: nonNull(userInputName),
     },
 
-    description: 'Update an existing user',
+    description: 'Delete an existing user',
```

Comment:
I think you renamed the wrong description you are in the update-user.js file instead of the delete-user.js file :)

Reply:
Thanks :) Fixed